synapse/rest/client/v2_alpha/account.py

```diff
@@ -442,9 +472,38 @@ def on_POST(self, request):
         if existing_user_id is not None:
             raise SynapseError(400, "Email is already in use", Codes.THREEPID_IN_USE)
 
-        ret = yield self.identity_handler.requestEmailToken(
-            id_server, email, client_secret, send_attempt, next_link
-        )
+        if self.config.threepid_behaviour_email == ThreepidBehaviour.REMOTE:
+            # Have the configured identity server handle the request
+            if not self.hs.config.account_threepid_delegate_email:
```

Comment:
is it possible for `threepid_behaviour_email == ThreepidBehaviour.REMOTE` without `account_threepid_delegate_email` being set? This would be better as an `assert` if not.

Reply:
It's not, no. Good call.